### PR TITLE
Changes to display control-Service-status. Test-case for conrol statusmonitor

### PR DESCRIPTION
**Pull request by tmallick**
Made changes to use control-node introspect instead of analytics server uves to display status of control-node in statusmonitor. Also added UT test-case for control-statusmonitor

### CODE REVIEW

File: statusmonitor/main.go [hunk 1]
```diff
@@ -170,36 +193,41 @@ func CreateRestClient(config Config) (http.Client, error) {
...
 				bodyBytes, err := ioutil.ReadAll(resp.Body)
+				bodyBytes_p, err_p := ioutil.ReadAll(process_resp.Body)
 				if err != nil {
 					log.Println(err)
 					return err
 				}
-				controlStatus := getControlStatusFromResponse(bodyBytes)
+				if err_p != nil {
+					log.Println(err_p)
+					return err_p
```
Comment (vijayrap) [inline statusmonitor/main.go:228]: pls eloborate the error message

File: statusmonitor/main.go [hunk 2]
```diff
@@ -399,57 +428,41 @@ func updateControlStatus(config *Config, controlStatusMap map[string]contrailOpe
...
 	}
-	connectionList := []contrailOperatorTypes.Connection{}
 
-	bgpRouterList := typeSwitch(controlUVEStatus.BgpRouterState.BgpRouterIPList)
-	bgpRouterConnection := contrailOperatorTypes.Connection{
-		Type:  "BGPRouter",
-		Nodes: bgpRouterList,
+	err = xml.Unmarshal(statusBody_p, &processst)
+	if err != nil {
+		log.Fatal(err)
```
Comment (vijayrap) [inline statusmonitor/main.go:442]: remove Fatal and use Print. with log

**Comment by Dysproz** [comment]
recheck ha

File: statusmonitor/main.go (hunk 2 above)
Comment (tmallick) [reply statusmonitor/main.go]: Added changes .

File: statusmonitor/main.go (hunk 1 above)
Comment (tmallick) [reply statusmonitor/main.go:228]: Added changes

**Review by Dysproz — APPROVED** [review]

**Pull request merged**